src/main/java/com/aws/greengrass/certificatemanager/certificate/CISShadowMonitor.java

```diff
@@ -145,6 +149,7 @@ public void stopMonitor() {
      * @param certificateGenerator CertificateGenerator instance for the certificate
      */
     public void addToMonitor(CertificateGenerator certificateGenerator) {
+        LOGGER.atInfo().log("Add server certificate generator to the watcher queue");
```

Comment:
debug? I think we need additional information for this to be useful, such as caller identity

Reply:
Yes, need more info on the caller identity to make it useful. But I would like it to be info so we can trace the sequences in case of some rare timing issue happening.